snowpack/src/build/build-import-proxy.ts

```diff
@@ -75,7 +77,8 @@ export function wrapHtmlResponse({
   });
 
   if (hmr) {
-    const hmrScript = `<script type="module" src="${getMetaUrlPath('hmr.js', config)}"></script>`;
+    const setHmrUrlScript = hmrUrl ? `<script type="text/javascript">window.HMR_WEBSOCKET_URL="${hmrUrl}";</script>\n` : '';
```

Comment:
no need for this, we already have a section in our docs telling users how to configure this themselves outside of Snowpack.

Reply:
Our default port is 12321. how to inform users what is url of websocket if we don't set HMR_WEBSOCKET_URL by ourselves?

Reply:
There's definitely still work to be done here, but I'd rather do it all together in a follow up PR and leave this existing limitation in place for now.

The reason is that this was always a bandaid, and now we may just want to move this entirely into a Snowpack config value, and then remove the `window. HMR_WEBSOCKET_URL ` snippet entirely.

Because ESM-HMR is shared we can't remove `window.HMR_WEBSOCKET_URL` entirely, but Snowpack could inject it into the `hmr.js` file itself, instead:

```
// hmr.js

// injected by Snowpack during dev & build, if config value is set
window.HMR_WEBSOCKET_URL = ${"VALUE FROM SNOWPACK CONFIG}

// the static hmr.js file contents
...
```